client/js/socket-events/connection.js

```diff
@@ -42,6 +42,11 @@ function handleDisconnect(data) {
 	store.commit("currentUserVisibleError", `Waiting to reconnect… (${message})`);
 	updateLoadingMessage();
 
+	if (message === "xhr poll error" && store.state.serverConfiguration.headerAuthEnabled) {
+		socket.disconnect();
+		window.location.reload(true);
```

Comment:
Is this being done because when we try to reconnect the auth header will no longer be included, so we need to reload the page to reconnect?

Reply:
Yep. There's no easy cross-platform way to do this.